lib/mayaUsd/ufe/UsdAttribute.cpp

```diff
@@ -177,7 +177,11 @@ U getUsdAttributeVectorAsUfe(
 {
     VtValue vt;
     if (!attr.isValid() || !attr.hasValue()) {
-        vt = MayaUsd::ufe::vtValueFromString(attr.typeName(), attr.defaultValue());
```

Comment:
Do not crash if there is no default value. And do even report an error. These are problems coming from projects out of the control of Autodesk, so there is not much the end user can do about them.